pages/common/ctags.md

```diff
@@ -0,0 +1,16 @@
+# ctags
+
+> Generates an index (or tag) file of language objects found in source files for many popular programming languages.
+> More information: <https://ctags.io/>
+
+- Generate tags for a single file, and save them in a file named "tags" in the current directory:
+
+`ctags {{path/to/file}}`
+
+- Generate tags for all files in the current directory and save them in a file with a custom filename:
+
+`ctags -f {{filename}} *`
+
+- Generate tags for all files in the current directory and all subdirectories (-R, --recurse):
+
+`ctags -R`
```

Comment:
```suggestion
`ctags --recurse`
```
It's not `--recursive`, but `--recurse`, right?

Reply:
Yes, it's `--recurse`. My thought process with this example was that it's much more common for people to use the short form (`-R`) in both scripts and commands, so I figured it would be better to give that form as the example, while explaining the full meaning in the description. 

Do you think it's better to just give the long form in the example, and not mention the short form?

Reply:
In this project it's preferred to use the longer version if it makes things more clear. I'm neutral on that, but I'd leave it here up for the discussion.

Reply:
I guess it depends on how often this command is used.

We could potentially use the short version if we use the word `recurse` in the description, but I'd probably say that it's best to use the long-form argument here to be consistent with other pages.

Reply:
Sure, I know that's the convention for tldr pages, which is why I was debating adding it. Another reason for including the short option that I considered is that usually, you can guess the short option from the long option (e.g. for the `--recursive` option for other tools, the short option will usually be `-r`) . Here that's not the case, as the short option is not `-r`, but `-R`.

Still, if the general agreement is that it doesn't add any value to have it in the tldr page, I'll remove it.

Reply:
I see. Generally we'll use a short option if we can somehow include the letter in the description text (e.g. `- [R]ecurse over foo:`).

I think in this instance we should use the long option.